pallets/stream/src/lib.rs

```diff
@@ -492,6 +575,8 @@ pub mod pallet {
 
 			<Streams<T>>::take(&stream_id);
 
+			<Commits<T>>::take(&stream_id);
```

Comment:
This is not correct. This will erase the complete history. We need to read from the commit and iterate the result to remove the entries from unique proofs.